flink-connectors/flink-connector-rabbitmq/src/main/java/org/apache/flink/streaming/connectors/rabbitmq/common/RMQConnectionConfig.java

```diff
@@ -132,8 +141,12 @@ private RMQConnectionConfig(
             Integer requestedChannelMax,
             Integer requestedFrameMax,
             Integer requestedHeartbeat,
-            Integer prefetchCount) {
+            Integer prefetchCount,
+            Long deliveryTimeout) {
         Preconditions.checkNotNull(uri, "Uri can not be null");
+        Preconditions.checkArgument(
+                deliveryTimeout == null || deliveryTimeout >= 0,
+                "deliveryTimeout can not be negative");
```

Comment:
Same comment here about splitting the checks or updating the message.

Reply:
Same comment as above.